ReadMe.md

```diff
@@ -1,18 +1,21 @@
 # Hello-Microverse_Project
 
-Hello-Microverse_project is a beginner level project at Microverse, that requires requires the student to display hello on a web page with simple styling. 
+Hello-Microverse_project is a beginner level project at Microverse that requires the student to add "Hello Microverse! to a web page with simple styling. 
 
 # Built With
 HTML
 CSS
 
 # Key Features
-1. Linter for HTML and CSS: checks errors and organise code
-2. gitignore: to keep some directories from being committed
+1. Linter for HTML and CSS: checks errors and organizes code
+2. gitignore: keeps selected directories from being committed
 
 # Prerequisites
 In order to run this project you need: a code editor
 
```

Comment:
- [ ] Please update your README file using [this updated template](https://github.com/microverseinc/readme-template) 😉

Reply:
Hi @hiromataba,
The Read Me file has been updated using the provided template. Thank you.